### PR TITLE
feat: Generate types for cosmwasm

### PR DESCRIPTION
**Pull request by apollo-sturdy**
The types in `cosmos-rust` don't have serde types derived, but these do!

### CODE REVIEW

File: packages/osmosis-std/src/types/cosmos/base/v1beta1.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ use osmosis_std_derive::CosmwasmExt;
     ::schemars::JsonSchema,
     CosmwasmExt,
 )]
-#[proto_message(type_url = "/cosmos.base.v1beta1.CoinCoin")]
```
Comment (ibjc) [inline packages/osmosis-std/src/types/cosmos/base/v1beta1.rs:17]: these type urls seem off (both before and after)

Comment (iboss-ptk) [reply packages/osmosis-std/src/types/cosmos/base/v1beta1.rs:17]: totally.. wondering why.. gotta find time to have a look

Comment (apollo-sturdy) [reply packages/osmosis-std/src/types/cosmos/base/v1beta1.rs:17]: Pushed a fix for this just now. Not sure if this is the right approach as I'm not familiar with how the file descriptors work, but it seems to work. Let me know what you think.

Comment (apollo-sturdy) [reply packages/osmosis-std/src/types/cosmos/base/v1beta1.rs:17]: Pushed a fix for this just now. Not sure if this is the right approach as I'm not familiar with how the file descriptors work, but it seems to work. Let me know what you think.

**Comment by iboss-ptk** [comment]
This looks awesome! Thanks @apollo-sturdy !

**Pull request merged**